docs/installation.rst

```diff
@@ -44,6 +58,10 @@ versions:
 .. _pip: https://github.com/pypa/pip
 .. _PyPI: https://pypi.org/
 
+.. _tox: https://tox.readthedocs.org/
+.. _virtualenv: https://virtualenv.readthedocs.org/
```

Comment:
virtualenv.pypa.io is the official link

Reply:
Thank you very much for the review @gaborbernat. I changed the link according to your comment.